code/modules/hydroponics/hydro_tray.dm

```diff
@@ -387,6 +387,11 @@
 	else if((O.sharpness_flags & (SHARP_BLADE|SERRATED_BLADE)) && harvest)
 		attack_hand(user)
 
+	else if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/grown)) //composting
+		to_chat(user, "You use [O] as compost for [src].")
```

Comment:
"you use \the [O] as compost for \the [src]."

Reply:
Done.